pkg/services/star/api/api.go

```diff
@@ -1,22 +1,70 @@
 package api
 
 import (
+	"context"
 	"net/http"
 	"strconv"
 
 	"github.com/grafana/grafana/pkg/api/response"
+	"github.com/grafana/grafana/pkg/api/routing"
```

Comment:
It would be nice to remove `routing.Wrap` as part of such PRs and use plain old http handlers instead